features/keymap/ncl/layers-set_layers_to.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Layers (Set Layers To)
```

Comment:
"Layer Modification, Set Layers To"?

I think "hold" makes sense as the modifier for demonstrating layering functionality. But, as a separate spec, I think this feature can be specified with a clearer name.